README.md

```diff
@@ -1,70 +1,75 @@
-# Getting Started with Create React App
+<p align="left">
+  <a href="https://www.microverse.org/">
+    <img alt="Microverse" src="https://img.shields.io/badge/-Microverse-blueviolet?style=flat-square">
+  </a>
+</p>
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Covid-Stats-App
+Covid-tracker is a website that allows users to have access to the latest covid data in Brazil. They can also filter by region name, as well as get more informations on that specific region by clicking on it.
 
-## Available Scripts
+| Countries Page | Details Page |
+| --- | --- |
+| ![screenshot](./countries.PNG) | ![screenshot](./details.PNG) |
 
-In the project directory, you can run:
+## Built With
 
-### `npm start`
+- HTML, CSS, React & Redux;
+- Visual Studio, Git & GitHub;
+- [Narrativa API](https://covid19tracking.narrativa.com/index_en.html).
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## Getting Started
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+### Running locally
+To run this project, you only need a computer with a browser (like Google Chrome, Mozilla Firefox, Microsoft Edge, Apple Safari, etc.) installed, and follow these steps:
 
-### `npm test`
+1. In your terminal, type the following, to clone this repository:
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+```sh
+$ git clone https://github.com/Cathella/covid-stats-app.git
+```
 
-### `npm run build`
+2. After, run the following commands in your terminal:
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+```sh
+$ cd covid-stats-app
+$ npm install
+$ npm start
+```
+<details align="right">
+<summary><small>What are these commands?</summary>
+- the `$ cd` command is used to move to different folders. <br>
+- `$ npm install` install all the required dependencies to run the project.
+- while `$ npm run start` runs the app in the development mode.</small>
+</details>
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+3. If your default browser do not start automatically, open it and type `http://localhost:3000` to run the application.
 
-### `npm run eject`
+### Running tests
+To launch the test runner in the interactive watch mode, run `$ npm test` in your console.
+> To read more information about running tests, visit [create react app website](https://create-react-app.dev/docs/running-tests/);
```

Comment:
- [ ] There are not enough tests present in the project. There is one test and that is failing. 
<img width="331" alt="Capture" src="https://user-images.githubusercontent.com/58504655/148643383-7d2f2744-e0ff-495a-840d-a0c098355dfb.PNG">


- Make sure app has unit tests for several components of the app and some pure functions in Redux code.
- App has integration tests using React Testing Library
   * You can perform a mocking data fetching and Redux Store connection.